combo_cases_and_deaths/delphi_combo_cases_and_deaths/run.py

```diff
@@ -42,23 +50,55 @@ def maybe_append(df1, df2):
         return df1
     return df1.append(df2)
 
-COLUMN_MAPPING = {"time_value": "timestamp",
-                  "geo_value": "geo_id",
-                  "value": "val",
-                  "stderr": "se",
-                  "sample_size": "sample_size"}
+
+def compute_special_geo_dfs(df, signal, geo):
+    """Compute the signal values for special geos (HHS and nation).
+
+    For `num` signals, just replace the geocode to the appropriate resolution.
+    For `prop` signals, replace the geocode and then compute the proportion using the total
+    population of the us.
+
+    Parameters
+    ----------
+    df: DataFrame
+        Dataframe with num values at the county level.
+    signal: str
+        Signal name, should end with 'num' or 'prop'.
+    geo: str
+        Geo level to compute.
+    Returns
+    -------
+        DataFrame mapped to the 'geo' level with the correct signal values computed.
+    """
+    df = GMPR.replace_geocode(df,
+                              from_col="geo_id",
+                              from_code="fips",
+                              new_code="state_code",
+                              date_col="timestamp")
+    df = GMPR.add_population_column(df, "state_code")  # use total state population
+    df = GMPR.replace_geocode(df, from_code="state_code", new_code=geo, date_col="timestamp")
```

Comment:
I think this logic works here, since megacounty values will be included and then we can just assume all the counties are reporting.